### PR TITLE
Tag hoverover truncation + Tag Page typography update

### PR DESCRIPTION
**Pull request by Raemon**
image

### CODE REVIEW

File: packages/lesswrong/components/tagging/TagPreview.tsx [hunk 1]
```diff
@@ -25,22 +26,24 @@ const TagPreview = ({tag, classes}: {
...
     ssr: true,
   });
-  
+
+  const highlight = truncate(tag.description?.htmlHighlight, 1, "paragraphs")
+
   return (<div>
-    <h2 className={classes.tagTitle}>{tag?.name} Tag</h2>
+    <h2 className={classes.tagTitle}>{tag.name} Tag</h2>
     {tag && <ContentItemBody
```
Comment (Raemon) [inline packages/lesswrong/components/tagging/TagPreview.tsx:44]:
```suggestion
    <ContentItemBody
```

File: packages/lesswrong/components/tagging/TagPreview.tsx [hunk 2]
```diff
@@ -25,22 +26,24 @@ const TagPreview = ({tag, classes}: {
...
   return (<div>
-    <h2 className={classes.tagTitle}>{tag?.name} Tag</h2>
-    {tag && <ContentItemBody
+    <h2 className={classes.tagTitle}>{tag.name} Tag</h2>
+    <ContentItemBody
       className={classes.tagDescription}
-      dangerouslySetInnerHTML={{__html: tag.description?.htmlHighlight}}
+      dangerouslySetInnerHTML={{__html: highlight}}
       description={`tag ${tag.name}`}
     />}
```
Comment (Raemon) [inline packages/lesswrong/components/tagging/TagPreview.tsx:48]:
```suggestion
    />
```

**Comment by Discordius** [comment]
Can you share a screenshot of what the hoverover UI looks like for the rationality tag?

**Comment by Raemon** [comment]
It's the same, just crops to the first paragraph

image

**Comment by Raemon** [comment]
Now also makes the Tag Hover and TagRelCard use the title as a fallback if there isn't a description

image

**Comment by Raemon** [comment]
image

**Pull request merged**